### PR TITLE
S/issues

### PR DESCRIPTION
**Pull request by stevenweber**
01-issue-template
02-create-issue
03-slack

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/console-write-access.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+name: Console Write Access
```
Comment (mcostanza) [inline .github/ISSUE_TEMPLATE/console-write-access.md:2]: This file is the template for the Console Write Access / Get Started screenshot in the PR description right?

File: src/on_issue.ts [hunk 2]
```diff
@@ -1,3 +1,107 @@
...
+  WebhookPayloadIssues,
+} from '@octokit/webhooks';
+
+import { postMessage } from './slack';
+
+const LABELED_ACTION = 'labeled';
+const NEWLINE = '\n';
+const RELEVANT_LABELS = getInput('relevant_labels').split(',');
+
+export async function onIssue(context) {
```
Comment (mcostanza) [inline src/on_issue.ts:17]: 👍

File: .github/ISSUE_TEMPLATE/console-write-access.md (hunk 1 above)
Comment (stevenweber) [reply .github/ISSUE_TEMPLATE/console-write-access.md:2]: Yup!

Comment (ebenoist) [reply .github/ISSUE_TEMPLATE/console-write-access.md:2]: This is great-- will this template be accessible via the action or do we have to check it in separately?

File: .github/workflows/sample.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
-name: sample-workflow
+name: sample-breakglass
```
Comment (ebenoist) [inline .github/workflows/sample.yml:1]: 👍

File: src/on_issue.ts [hunk 4]
```diff
@@ -1,3 +1,107 @@
...
+  } = repository;
+
+  const {
+    html_url,
+    number,
+    body,
+  } = issue;
+
+  const quotedBody = body.split(NEWLINE).map(line => `> ${line}`).join(NEWLINE);
+  await record(`<${html_url}|#${number}> (${name}) _*${labelName}*_ requested by ${actor}\n\n${quotedBody}`);
```
Comment (ebenoist) [inline src/on_issue.ts:78]: Nice touch

File: src/slack.ts [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import * as request from 'request-promise-native';
+import * as core from '@actions/core';
+
+const hook = core.getInput('slack_hook', {
+  required: true,
+});
+
+export async function postMessage(text: string) {
```
Comment (ebenoist) [inline src/slack.ts:8]: 👍

File: .github/ISSUE_TEMPLATE/console-write-access.md (hunk 1 above)
Comment (stevenweber) [reply .github/ISSUE_TEMPLATE/console-write-access.md:2]: I don't believe this can be automatically hooked in, pros/cons I guess.

**Pull request merged**